playground/backend/internal/utils/cache_utils.go

```diff
@@ -32,3 +33,14 @@ func SetToCache(ctx context.Context, cacheService cache.Cache, key uuid.UUID, su
 	}
 	return err
 }
+
+// ReadAndSetToCacheGraph read graph from file and writes it to the cache.
+func ReadAndSetToCacheGraph(ctx context.Context, cacheService cache.Cache, pipelineId uuid.UUID, graphPath string) error {
```

Comment:
Could you split this function? This implementation does not follow the single responsibilities principle. It will be better to read files outside this function.

Reply:
Done